requirements.txt

```diff
@@ -1,3 +1,4 @@
+cohere<5.0.0a0
```

Comment:
But cohere should not be a main requirement, please remove this line! 
it can be a requirement in setup.py if you use [all]